dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops

```diff
@@ -36,6 +36,30 @@ SELECTION_PAGE_PASSWORD=Password
 
 SELECTION_PAGE_SUBMIT=Submit
 
+# 0=Social login configuration attribute name, 1=ID of the social login configuration element, 2=Default value for the configuration attribute
+SOCIAL_LOGIN_CONFIG_ATTRIBUTE_EMPTY=CWWKS5370W: The value for the [{0}] configuration attribute in the [{1}] social login configuration is empty. The default value [{2}] is used.
```

Comment:
Change
"The value for the [{0}] configuration attribute in the [{1}] social login configuration is empty. The default value [{2}] is used."
to
"The value for the [{0}] configuration attribute in the [{1}] social login configuration is empty. The default value [{2}] for the configuration attribute is used."